### ISSUE TITLE
JDKPackagerPlugin can't package ScalaFX application 

### ISSUE THREAD

**Issue by strobe** [sbt/sbt-native-packager#563]
Got this error even if main class was set at build.sbt manually (on OSX).

```
[error] Exception in thread "main" com.sun.javafx.tools.packager.PackagerException: Error: Unknown argument: hello.ScalaFXHelloWorld
[error] 	at com.sun.javafx.tools.packager.Main.main(Main.java:340)
```

as example of ScalaFX app see https://github.com/scalafx/scalafx-hello-world/blob/master/src/main/scala/hello/ScalaFXHelloWorld.scala

**Comment by muuki88** [comment]
Is this a sbt-native-packager error? For me it looks more like a general sbt/javafx missconfiguration?

cc @metasim

**Comment by metasim** [comment]
@strobe What version of Java are you using?

**Comment by strobe** [comment]
I'm using this
```
java version "1.8.0_45"
Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)
```

I created simple project for test this issue 

https://github.com/strobe/ScalaFxPackaging

**Comment by metasim** [comment]
@strobe Thanks a lot. Will take a look at it soon.

**Comment by metasim** [comment]
I don't think the bug has anything to do with ScalaFX, but with the plugin improperly handling the case when no maintainer is specified.

@strobe  Working on a fix, but try setting the `maintainer` key and see if you get a package out of it.

@muuki88 Could you please change the title of this issue to: "JDKPackagerPlugin fails to build package if `maintainer` is not set."?

**Comment by strobe** [comment]
@metasim
checked - yes it working well if maintainer was set

**Comment by metasim** [comment]
PR submitted.

**Comment by muuki88** [comment]
fix with #566
